applicants.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Applicants</title>
+    <link rel="stylesheet" href="./src/applicants.css">
+    <link rel="stylesheet" href="./src/main.css">
+</head>
+<body>
+    <header>
+        <h1>
+            <img class="waves" src="./assets/wave.jpg" alt="image of waves">
+            <p>
```

Comment:
No need for this p tag here. If 'Applicants' is the title of the page then it should be wrapped by the h1. Screen readers and search engines look for h1's to determine content, not p tags :-)
```html
<h1>
    <img class="waves" src="./assets/wave.jpg" alt="image of waves">
    Applicants
</h1>
```